docs/user-guide/admin-cli.md

```diff
@@ -0,0 +1,141 @@
+[TOC]
+
+# Introduction
+In the context of the docker volume plugin for vSphere, each ESXi host manages multiple VMs, with
+each of them acting as a docker host. The docker engine on these hosts communicates with the docker
+volume plugin in order to create, delete and mount volumes. These volumes may live on any datastore
+accessible to the ESXi host and are managed by the docker user via the docker CLI. However, the
+docker CLI is limited in what visibility it can provide to the user. Furthermore, it is desirable
+that an administrator be able to get a global view of all volumes created and in use on the host.
+For these reasons, an admin CLI has been created that runs on the ESXi host and that provides access
+to information not visible from the docker CLI. The remainder of this document will describe each
+admin CLI command and provide examples of their usage.
+
+
+# ls
```

Comment:
This brings another question (not related to the doc) - do you think we should wrap it all in esxcli command? Esxcli  is easily expandable - an XML file maps esxcli subcommands to external command line